topological_navigation/launch/topological_navigation.launch

```diff
@@ -37,6 +38,8 @@
 	<node pkg="topological_navigation" name="execute_policy_server" type="execute_policy_server.py" output="screen" respawn="true">
 		<param name="retries" type="int" value="$(arg execute_policy_retries)"/>
 		<param name="move_base_reconf_service" type="str" value="$(arg move_base_reconf_service)"/>
+		<param name="move_base_planner" type="str" value="$(arg move_base_reconf_service)"/>
```

Comment:
either the `value="$(arg move_base_reconf_service)"`for the param `move_base_planner` 
or
if the value of `move_base_planner` is same as that of `move_base_reconf_service` the new argument may not be needed.